source/getting-started/faq.rst

```diff
@@ -351,6 +352,33 @@ ________________________________________________________________
 
 cc-snapshot is occasionally updated to accommodate changes to the infrastructure and distributions. To replace the script in your image, follow our instructions for :ref:`updating cc-snapshot <updating-snapshot>`.
 
+.. _move-images-between-sites:
+
+How to move images between sites?
+_____________________________________
+
+Chameleon bare-metal sites  -- ``CHI@TACC`` and ``CHI@UC`` -- are two regions of the same but :ref:`independent <bare-metal-sites-independent>` OpenStack deployment. 
```

Comment:
@priteau I have "the same BUT independent OpenStack deployment".

Reply:
I don't think it's understandable for users. I would say:

    Chameleon bare-metal sites -- ``CHI@TACC`` and ``CHI@UC`` -- belong to a single OpenStack deployment as two :ref:`independent <bare-metal-sites-independent>` regions.